packages/oidc-middleware/src/ExpressOIDC.js

```diff
@@ -36,6 +34,7 @@ module.exports = class ExpressOIDC extends EventEmitter {
    * @param {string} [options.scope=openid] The scopes that will determine the claims on the tokens
    * @param {string} [options.response_type=code] The OpenId Connect response type
    * @param {number} [options.maxClockSkew=120] The maximum discrepancy allowed between server clocks in seconds
+   * @param {Object} [options.skipConfigValidation] Skip configuration validation for specific keys
```

Comment:
I think we should use `testing.disableHttpsCheck` to be consistent with what we've developed for the management SDKs.

Reply:
I originally thought about this, but ultimately decided against it.

In the future, its likely that we will want to skip `clientId` and `clientSecret` validation, as these may not be known when initializing the `ExpressOIDC` object (Dynamic client registration). Nesting the flags under `testing` would not apply to only testing use cases.

`skipConfigValidation` may not be the best name, but I see it as verbose enough to tell users of this library that they're going against the norm by using this object.

Reply:
We had a long conversation about this, and in the end we decided this should go under the `testing` key to be very clear that this is for a testing case and should be avoided for any kind of real production situation.  Hence the warning I mention above.  We could still add some other key in the future to address the client ID concerns you raise, but we'd still want to break out this https disabling to the `testing` key.